QUICKSTART-EKS.md

```diff
@@ -158,11 +158,11 @@ aws ssm get-parameters --region us-west-2 \
 
 ### Cluster setup
 
-*Note:* most commands will have a region argument; make sure to change it if you don't want to set up in us-west-2. 
-Also be aware that when operating in GovCloud the IAM ARNs will need to be updated to the following: `arn:aws-us-gov`.   
-For example:  
- `arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy`   
- will be updated to:  
+*Note:* most commands will have a region argument; make sure to change it if you don't want to set up in us-west-2.
+Also be aware that when operating in GovCloud the IAM ARNs will need to be updated to the following: `arn:aws-us-gov`.
+For example:
+ `arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy`
+ will be updated to:
```

Comment:
nit: I like the cleanup but I tend to want it in a separate commit - can still be in this PR though